src/components/form/GridButton.jsx

```
@@ -5,18 +5,25 @@ import { useSelector } from "react-redux";
-        size={20}
+    <div className="grid-button bg-gray-200 rounded-lg">
+      <div
         onClick={onListView}
-        className={`hover:opacity-60 ${!grid && textColor}`}
-      />
-      <IoGridOutline
-        size={20}
+        className={`flex items-center justify-center w-8 h-8 rounded-lg cursor-pointer transition-opacity duration-200 
+                    ${!grid ? 'bg-white text-black' : 'bg-transparent'} 
+                    hover:opacity-60`}
+      >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)